src/sentry/static/sentry/app/components/avatar/index.jsx

```diff
@@ -1,16 +1,20 @@
+import PropTypes from 'prop-types';
 import React from 'react';
 
-import SentryTypes from 'app/sentryTypes';
-import TeamAvatar from 'app/components/avatar/teamAvatar';
 import OrganizationAvatar from 'app/components/avatar/organizationAvatar';
 import ProjectAvatar from 'app/components/avatar/projectAvatar';
+import SentryTypes from 'app/sentryTypes';
+import TeamAvatar from 'app/components/avatar/teamAvatar';
 import UserAvatar from 'app/components/avatar/userAvatar';
 
+const BasicModelShape = PropTypes.shape({slug: PropTypes.string});
```

Comment:
Is this worth having on sentryTypes? Perhaps as BasicAvatarShape?

Reply:
Probably not unless we have other components that accept these models progressively (e.g. slug -> full model)